types/config/config.go

```diff
@@ -49,7 +50,7 @@ func DefaultConfig() Config {
 }
 
 func (c Config) GetBytes() ([]byte, error) {
-	return c.bytes, nil
+	return yaml.Marshal(c)
```

Comment:
This will break everything when using the `GetBytes` method on other projects. Instead, we should fix the initialization inside the `DefaultConfig` if the problem is there

Reply:
Updated it.